nucypher/network/nodes.py

```diff
@@ -692,8 +725,7 @@ def learn_from_teacher_node(self, eager=True):
 
 
 class Teacher:
-    TEACHER_VERSION = 1
-    _evidence_of_decentralized_identity = constants.NOT_SIGNED
+    TEACHER_VERSION = LEARNING_LOOP_VERSION
```

Comment:
Just to be sure this is intentional; This will create two separate references, and incrementing the constant will not effect a running teacher:

```
>>> LEARNING_LOOP_VERSION = 1
>>> class Teacher:
...     TEACHER_VERSION = LEARNING_LOOP_VERSION
... 
>>> Teacher.TEACHER_VERSION
1
>>> LEARNING_LOOP_VERSION = 2
>>> Teacher.TEACHER_VERSION
1
```

Reply:
I don't imagine ever changing this value during runtime... Maybe I'll use constantSorrow for it?